lib/synced/strategies/full.rb

```diff
@@ -183,7 +184,7 @@ def api_request_options
             options[:include] += @include
           end
           options[:fields] = @fields if @fields.present?
-          options[:auto_paginate] = true
+          options[:auto_paginate] = true if @auto_paginate
```

Comment:
You're right, when writing I already had in mind that it is just temporary solution and will be changed shortly. Wouldn't hurt to correct it though.